python/cugraph/pyproject.toml

```diff
@@ -32,6 +32,7 @@ dependencies = [
     "libcugraph==25.6.*,>=0.0.0a0",
     "numba>=0.59.1,<0.61.0a0",
     "numpy>=1.23,<3.0a0",
+    "pylibcudf==25.6.*,>=0.0.0a0",
```

Comment:
This change should be reflected in the `cugraph` conda recipe as well:

Add `pylibcudf` to both the `host:` and `run:` environments, similar to how `pylibraft` is handled:

https://github.com/rapidsai/cugraph/blob/c3c19e39070638c1bc03320990772cb9d4c60cb2/conda/recipes/cugraph/recipe.yaml#L63

That dependency is getting satisfied transitively because it comes through `cudf`, but now that you're introducing a direct usage of `pylibcudf`, we should make that explicit.

Reply:
Ah sure thing. Yeah it's easy to forget this since it's not modified by the dependency file generator

Reply:
Yeah for sure. We're hoping that now that we're using `rattler-build`, it might be possible soon to have `rapids-dependency-file-generator` update these files.

In short... the main challenge was that all the Jinja templating stuff in `conda-build` recipes meant that they weren't valid YAML, which made automatic updates hard. `rattler-build` recipes are guaranteed to be valid YAML files 😁 

@vyasr wrote up a nice summary here: https://github.com/rapidsai/dependency-file-generator/issues/7#issuecomment-2766675431